src/appmixer/pipedrive/crm/CreateLead/CreateLead.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/**
+ * CreateLead action.
+ * @extends {Component}
+ */
+module.exports = {
+    async receive(context) {
+        let { title, ownerId = '', organizationId, amount, currency, visibleTo = '' } = context.messages.lead.content;
```

Comment:
Add a newline after function declaration.
```suggestion
    async receive(context) {

        let { title, ownerId = '', organizationId, amount, currency, visibleTo = '' } = context.messages.lead.content;
```